specs/ExtendedAcceleratorKeyPressed.md

```diff
@@ -0,0 +1,187 @@
+# Background
+WebView2 has `ICoreWebView2AcceleratorKeyPressedEventArgs` for the 
+`CoreWebView2Controller.AcceleratorKeyPressed` event. We have been asked to extend this API so
+that developers can indicate that theywould like an existing browser accelerator key to be 
+disabled as a browser accelerator.
+
+In this document we describe the updated API. We'd appreciate your feedback.
+
+# Description
+We propose extending the `ICoreWebView2AcceleratorKeyPressedEventArgs` to allow developers
+to enable/disable the browser from handling accelerator keys with `AllowBrowserAcceleratorHandling` 
+property. 
+
+# Examples
+## C++
+
+``` cpp
+    EventRegistrationToken m_acceleratorKeyPressedToken = {};
+
+    wil::com_ptr<ICoreWebView2Controller> m_controller;
+    wil::com_ptr<ICoreWebView2> m_webView;
+    wil::com_ptr<ICoreWebView2Settings3> m_settings3;
+
+  {

```

Comment:
```suggestion
   {
```
Believe it or not, I got confused by the mis-indentation.

Reply:
fixed!